muckrock/foiamachine/forms.py

```diff
@@ -45,6 +48,14 @@ class FoiaMachineCommunicationForm(forms.ModelForm):
     The FOIA Machine Communication form allows for creating and updating communications.
     Also allows files to be attached to the request.
     """
+
+    def __init__(self, *args, **kwargs):
+        super(FoiaMachineCommunicationForm, self).__init__(*args, **kwargs)
+        try:
+            self.initial['message'] = bleach.clean(self.initial['message'].replace("<div>", "").replace("</div>", "\n").replace("<br>", "\n"), strip=True)
+        except:
```

Comment:
Can you catch specific exceptions instead of a catch all?  I'm not sure which exceptions could be thrown here.